From e8ae09476c926295f2cf18f9aa8b7284d021c61e Mon Sep 17 00:00:00 2001
From: Joey <joey@lineageos.org>
Date: Tue, 3 Jul 2018 20:56:38 +0200
Subject: [PATCH 39/50] base: limit direct share targets to reduce lag

Limit target packages and services query to reduce lag
when opening the share menu

Change-Id: I00179a87132d5d186b9482e0b6424eae0d175fbd
Signed-off-by: Joey <joey@lineageos.org>
(cherry picked from commit 27fea0e96c09f7dbe3bf4d409a0219e8f568fe60)
---
 core/java/com/android/internal/app/ChooserActivity.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/com/android/internal/app/ChooserActivity.java b/core/java/com/android/internal/app/ChooserActivity.java
index c31f17af67a..80153e4c147 100644
--- a/core/java/com/android/internal/app/ChooserActivity.java
+++ b/core/java/com/android/internal/app/ChooserActivity.java
@@ -101,7 +101,7 @@ public class ChooserActivity extends ResolverActivity {
 
     private static final boolean DEBUG = false;
 
-    private static final int QUERY_TARGET_SERVICE_LIMIT = 5;
+    private static final int QUERY_TARGET_SERVICE_LIMIT = 3;
     private static final int WATCHDOG_TIMEOUT_MILLIS = 2000;
 
     private Bundle mReplacementExtras;
-- 
2.17.1

